packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -106,151 +289,148 @@ function WorklistDataGrid(props) {
         nextRow += 1;
         break;
       case KeyCode.KEY_LEFT:
-        nextCol -= 1;
+        if (event.metaKey) {
+          // Cmd + Left = Home
```

Comment:
Does this formatting mean: `<Mac command> = <Windows command>`? Can we update the comment to be clearer if so?

Reply:
It was meant to indicate the equivalent windows keys used to achieve the same functionality. The comments were updated to make this clearer.